latency/Cargo.toml

```diff
@@ -34,37 +34,38 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 async-std = { version = "=1.9.0", features = ["unstable"] }
 async-trait = "0.1.42"
+futures = "0.3.12"
 env_logger = "0.9.0"
 log = "0.4.14"
 rand = "0.8.3"
 slab = "0.4.2"
 structopt= "0.3.21"
-zenoh = { git = "https://github.com/eclipse-zenoh/zenoh.git", tag = "0.5.0-beta.9", default-features = false, features = ["transport_tcp", "transport_udp"] }
-zenoh-util = { git = "https://github.com/eclipse-zenoh/zenoh.git", tag = "0.5.0-beta.9" }
+zenoh = { git = "https://github.com/eclipse-zenoh/zenoh.git", rev="f7309d6af9aa8c5c6e55e744fc68ea4f6f18e8e9", default-features = false, features = ["transport_tcp", "transport_udp"] }
```

Comment:
Well, I don't think so... In order to capture which commit (of zenoh) we used in this PR, we'd better keep it. For example, I can observe the performance drop issue on throughput since  I manually update the zenoh crate in Cargo.toml.
Each upgrade of the Cargo.toml forces users to check the program's consistency.

Reply:
Here I just aligned the version of commit that used by yuyuan.

Reply:
I will make a PR to upgrade _throughput/Cargo.toml_  since the latest commit fix the performance issue. So maybe @eddie9712 you can directly use the lastest commit of zenoh in your PR.

Reply:
Fixing the version is okay when the it is for one time use (for particular purpose). However, as a tool that will be used repeatedly, probably needing to edit the version everytime when we need to use it is not a good idea. What do you think?